### ISSUE TITLE
IDE doesn't show DDB tables

### ISSUE THREAD

**Issue by cibersheep** [daad-adventure-writer/NAPS#4]
Sorry, I'm back to bother you :D 

If I understood it correctly, the IDE would open a data base (a DDB, right?)
`python3 ide_pyqt.py ../AD/COZUMEL/PART1.DDB` will show a blank screen, no error message.

Edit: it looks empty but i think it loads the database

Trying to show any of the parts of the database will show an empty list
i.e. Descripciones de localidades, will show an empty, this time we got this error for each line:
```
Traceback (most recent call last):
  File "ide_pyqt.py", line 474, in data
    return mod_actual.lee_secs_ctrl (self.listaTextos[index.row()], QChar)
NameError: name 'QChar' is not defined
```

**Comment by DjMorgul** [comment]
This should be fixed on commit b48f47a9837de932178898bcb4632e16c2b58fe3

**Comment by cibersheep** [comment]
Weee... I can tables! :)
Thank you again
